src/github.com/getlantern/flashlight/pac.go

```diff
@@ -9,11 +10,13 @@ import (
 	"github.com/getlantern/filepersist"
 	"github.com/getlantern/pac"
 
-	"github.com/getlantern/flashlight/proxiedsites"
+	"github.com/getlantern/flashlight/ui"
```

Comment:
Sorry, but it's `pac.go` depends on a sub-package here. detour doesn't have any dependency on flashlight.